library/common/src/main/java/com/epmedu/animeal/common/data/repository/FeedingPointRepositoryImpl.kt

```diff
@@ -0,0 +1,51 @@
+package com.epmedu.animeal.common.data.repository
+
+import com.epmedu.animeal.common.data.enum.AnimalPriority
+import com.epmedu.animeal.common.data.enum.AnimalState
+import com.epmedu.animeal.common.data.model.FeedingPoint
+import com.epmedu.animeal.common.data.model.MapLocation
+import com.epmedu.animeal.foundation.switch.AnimalType
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import kotlin.random.Random
+
+class FeedingPointRepositoryImpl : FeedingPointRepository {
+
+    private val stubData: List<FeedingPoint> = List(25) { index ->
+        FeedingPoint(
+            index,
+            AnimalPriority.values()[Random.nextInt(0, AnimalPriority.values().size)],
```

Comment:
Could be simplified like this: AnimalPriority.values().random()

Reply:
I don't know what I was thinking. Done